### DISCUSSION TITLE
for_window [urgent=latest] focus not working

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5106] [Q&A] [answered]
I am using i3 with polybar. I am facing a weird problem. For example when I click on a link in thunderbird which is in workspace 1 it should automatically redirect me to workspace 2 where my firefox is already opened. I tried setting "for_window [urgent=latest] focus " to my i3 config and reloaded my config. It always works for the first time after I reload my i3 but then it doesn't work. Can anyone please help me ?

**Comment by ankurdhama** [comment, marked as answer]
Try: `focus_on_window_activation focus`

**Reply by ghost** [reply]
OH MY GOD THANK YOU SO MUCH!! IT WORKS!!
